integrations/jina/src/haystack_integrations/components/embedders/jina/text_embedder.py

```diff
@@ -13,16 +13,19 @@
 @component
 class JinaTextEmbedder:
     """
-    A component for embedding strings using Jina models.
+    A component for embedding strings using Jina AI models.
 
     Usage example:
     ```python
-    from jina_haystack import JinaTextEmbedder
+    import os
+    from haystack_integrations.components.embedders.jina import JinaTextEmbedder
 
-    text_to_embed = "I love pizza!"
+    os.environ("JINA_API_KEY") = "YOUR_JINA_API_KEY"
```

Comment:
As in the document embedder, we should remove that because it does not show how the component is used.